scripts/20210628_create-isOnboardedAcc/create-isOnboardedAcc.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable */
+
+/*
+This script creates a new key, isOnboardedAccount, on the smscounts db which indexes the msgSrvcId variable
+*/
+
+let formTwilioId = 'insert the form twilio id here'
+
+// == PRE-UPDATE CHECKS ==
+// Count the number of verifications
+db.getCollection('smscounts').count({ smsType: 'VERIFICATION' })
```

Comment:
we should get a count of the documents with/without our own twilio ID too, so we can check if the updated amounts are exactly the same too

Reply:
sure, will do